### PR TITLE
feat: added text_annotation to vision conversion

### PR DESCRIPTION
**Pull request by galz10**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-documentai-toolbox/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: google/cloud/documentai_toolbox/wrappers/document.py [hunk 1]
```diff
@@ -173,9 +174,15 @@ def _convert_to_vision_annotate_file_response(text: str, pages: List[page.Page])
         page_vision_annotation.text = _get_text_anchor_substring(
```
Comment (xixiong) [inline google/cloud/documentai_toolbox/wrappers/document.py:174]: Remove the page_vision_annotation?

File: tests/unit/test_document.py [hunk 2]
```diff
@@ -522,7 +522,11 @@ def test_convert_document_to_annotate_file_response():
 
     actual = doc.convert_document_to_annotate_file_response()
 
-    assert actual != AnnotateFileResponse()
+    assert actual.responses[0].text_annotations[0].description == "Invoice"
+    assert len(actual.responses[0].text_annotations) == 86
```
Comment (xixiong) [inline tests/unit/test_document.py:526]: Do we have more detailed tests? such as comparing the responses?

File: google/cloud/documentai_toolbox/converters/vision_helpers.py [hunk 3]
```diff
@@ -226,6 +227,52 @@ def _convert_document_token(
...
+    page_info: Current page information, including document page to be converted
+      , its text, and the position of reading cursor.
+
+  Returns:
+    A list of EntityAnnotations with descriptions and bounding box populated. A
+    EntityAnnotation has a word level information.
+  """
+  text_annotations = []
+  for token in page_info.page.tokens:
+    v = []
```
Comment (holtskinner) [inline google/cloud/documentai_toolbox/converters/vision_helpers.py]: Could you change this `v` list to use the `Vertex` type for Vision? Just to maintain type safety

**Review by holtskinner — APPROVED** [review]

File: google/cloud/documentai_toolbox/wrappers/document.py [hunk 4]
```diff
@@ -173,9 +174,15 @@ def _convert_to_vision_annotate_file_response(text: str, pages: List[page.Page])
...
+
+        full_text_annotation = _convert_document_page(page_info)
+        full_text_annotation.text = _get_text_anchor_substring(text, pages[page_idx].documentai_page.layout.text_anchor)
+        text_annotations = _generate_entity_annotations(page_info)
+
         responses.append(
             AnnotateImageResponse(
-                full_text_annotation=page_vision_annotation,
+                full_text_annotation=full_text_annotation,
+                text_annotations=text_annotations,
```
[the remark this answers is not in the document]
Comment (galz10) [reply google/cloud/documentai_toolbox/wrappers/document.py:187]: if you look at `AnnotateImageResponse` , `text_annotations` is a `List[EntityAnnotation]`. I don't know why it's called `text_annotations` and not `entity_annotations`.

File: tests/unit/test_document.py (hunk 2 above)
Comment (galz10) [reply tests/unit/test_document.py:526]: Added a comparison for the full `AnnotateFileResponse`

**Review by xixiong — APPROVED** [review]

**Pull request merged**